static/js/lib/constants.js

```diff
@@ -11,9 +11,15 @@ export const COURSE_AVAILABLE_NOW = "Available Now"
 export const COURSE_ARCHIVED = "Archived"
 export const COURSE_PRIOR = "Prior"
 
+const ocwPlatform = "ocw"
+const edxPlatform = "mitx"
 export const platforms = {
-  OCW: "ocw",
-  edX: "mitx"
+  OCW: ocwPlatform,
+  edX: edxPlatform
```

Comment:
Missed this on my first pass, but this needs a set of constants and a platform image for bootcamps as well.

Reply:
right I meant to ask about that but it slipped my mind - what should the value of the constant be?

Reply:
`"bootcamps"`